src/main/java/com/org/jenkins/custom/jenkins/distribution/service/generators/PackageConfigGenerator.java

```diff
@@ -70,7 +70,7 @@ private static JSONObject generateBuildSettings(final JSONObject buildSettings)
         dockerInfo.put("base", buildSettings.getString("base"));
         if (buildSettings.has("tag")) {
             dockerInfo.put("tag", buildSettings.getString("tag"));
-            dockerInfo.put("build", "true");
+            dockerInfo.put("build", "false");
```

Comment:
Dang, this sees like a pretty big change.... There should be a test that accompanies this to show that the problem existed and that the problem was solved by this particular change.

Reply:
This actually fails the testing at this time because the test expects "true". Setting docker to true actually fails, see #155 and [this comment in #141](https://github.com/jenkinsci/custom-distribution-service/pull/141#issuecomment-680014232).